### ISSUE TITLE
Alert and Action Sheet rules collide

### ISSUE THREAD

**Issue by dwieeb** [ionic-team/v4-migration-tslint#17]
The `ion-action-sheet-method-create-parameters-renamed` and `ion-alert-method-create-parameters-renamed` collide with each other.

Both rules are only using the AST, so it looks like they're just looking for a method call by the name of `create()` with an object that has `title` and `subTitle` keys. For both rules this is true, so they output the lint error for each other, resulting in the same message printed twice for a single offence:

```
ERROR: <path>/src/app/pages/home/home.page.ts[15, 7]: The title field has been replaced by header.
ERROR: <path>/src/app/pages/home/home.page.ts[15, 7]: The title field has been replaced by header.
ERROR: <path>/src/app/pages/home/home.page.ts[16, 7]: The subTitle field has been replaced by subHeader.
ERROR: <path>/src/app/pages/home/home.page.ts[16, 7]: The subTitle field has been replaced by subHeader.
```

**Comment by dwieeb** [comment]
The code in place today also could potentially result in false positives.

I looked into using the type checker, which would resolve both issues, but I think it's more trouble than it's worth. The type checker is much more complicated than the AST utilities, and less frequently used (so fewer docs, etc).
